DEPS

```diff
@@ -84,6 +84,119 @@ vars = {
 
   # Setup Git hooks by default.
   "setup_githooks": True,
+
+  # upstream URLs for third party dependencies, used in
+  # determining common ancestor commit for vulnerability scanning
+  "upstream_urls" : {
```

Comment:
(Non-blocking comment) Going forward, will this be manually updated?

Reply:
Yes this will be manually updated if dependencies are added/removed. The test will fail if a dependency is added but does not appear in this list, so that should help with a reminder to manually add here